package.json

```diff
@@ -9,7 +9,6 @@
   "files": [
     "release/**/*",
     "dist/**/*",
-    "package-lock.json",
```

Comment:
it is ignored anyways, also this file goes nowhere ~
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#files

```
Conversely, some files are always ignored:
...
package-lock.json (use [npm-shrinkwrap.json](https://docs.npmjs.com/cli/v9/configuring-npm/npm-shrinkwrap-json) if you wish it to be published)
```